src/components/Table/TableCell.js

```diff
@@ -0,0 +1,60 @@
+import PropTypes from 'prop-types';
+import styled, { css } from 'styled-components';
+import React from 'react';
+import theme from '../../theme';
+import { get } from '../../utils/utils';
+import { COLOR, COMMON, TYPOGRAPHY } from '../../utils/constants';
+import TableHeadContext from './TableHeadContext';
+
+
+const sharedStyle = css`
+    padding: 0 20px;
+    &:first-child {
+        padding-left: 0;
+    }
+
+    &:last-child {
+        padding-right: 0;
+    }
+
+    ${get('typography.table')};
```

Comment:
I think even after putting this in <Table>, I would still leave this here. (comment above about putting typography styles in Table)

Reply:
Read this after removing it, sorry. I think it would be redundant, no?